app/[locale]/academics/curricula/[code]/page.tsx

```diff
@@ -1,14 +1,207 @@
-import { WorkInProgressStatus } from '~/components/status';
+import Image from 'next/image';
+import Link from 'next/link';
+import { MdEmail, MdPhone } from 'react-icons/md';
+
+import Heading from '~/components/heading';
+import ImageHeader from '~/components/image-header';
+import {
+  Accordion,
+  AccordionContent,
+  AccordionItem,
+  AccordionTrigger,
+} from '~/components/ui';
+import { getTranslations } from '~/i18n/translations';
 import { courses, db } from '~/server/db';
 
 export async function generateStaticParams() {
   return await db.select({ code: courses.code }).from(courses);
 }
 
-export default function Curriculum({
-  params: { locale },
+export default async function Curriculum({
+  params: { locale, code },
 }: {
   params: { locale: string; code: string };
 }) {
-  return <WorkInProgressStatus locale={locale} />;
+  const text = (await getTranslations(locale)).Curriculum;
+  const courses = (await db.query.courses.findFirst({
+    where: (course, { eq }) => eq(course.code, code),
+    with: {
+      coordinator: {
+        with: {
+          person: {
+            columns: {
+              name: true,
+              telephone: true,
+              email: true,
+            },
+          },
+        },
+      },
+    },
+  }))!;
+
+  return (
+    <>
+      <ImageHeader
+        title={`${courses.code}\n${courses.title}`} // to break course code and title into two different lines
+        src={`courses/${courses.code}/image.png`}
+        className="whitespace-pre-line bg-neutral-300"
+      />
+
+      <main className="container mt-10">
+        <section className="md:flex md:space-x-10">
+          <section className="my-auto space-y-4 md:h-auto md:w-[60%]">
+            <section>
+              <h5 className="mb-2 flex space-x-3 align-baseline">
+                {text.prerequisites}:
+                {courses.prerequisites.map((prerequisite, index) => (
+                  <Link
+                    href={`/academics/curricula/${prerequisite}`}
+                    key={index}
+                    className="ml-2"
+                  >
+                    <p className="text-primary-300 underline">{prerequisite}</p>
+                  </Link>
+                ))}
+              </h5>
+            </section>
+
+            <section className="flex space-x-2 align-baseline">
+              <h5>{text.objectives}:</h5>
+              <p>{courses.objectives}</p>
+            </section>
+
+            <h5>
+              {text.nature}: <strong>{courses.nature}</strong>
+            </h5>
```

Comment:
`courses.nature` should be outside `h5`. I guess this would look something like:
```suggestion
            <h5>{text.nature}: </h5>
            <p>{courses.nature}</p>
```